src/controllers/movies/moviesrecommended.js

```diff
@@ -266,45 +280,46 @@ define(['events', 'layoutManager', 'inputManager', 'userSettings', 'libraryMenu'
         }
 
         function getTabController(page, index, callback) {
-            var depends = [];
+            let depends = '';
 
             switch (index) {
                 case 0:
-                    depends.push('controllers/movies/movies');
+                    depends = 'controllers/movies/movies';
                     break;
 
                 case 1:
+                    depends = 'controllers/movies/moviesrecommended.js';
                     break;
 
                 case 2:
-                    depends.push('controllers/movies/movietrailers');
+                    depends = 'controllers/movies/movietrailers';
                     break;
 
                 case 3:
-                    depends.push('controllers/movies/movies');
+                    depends = 'controllers/movies/movies';
                     break;
 
                 case 4:
-                    depends.push('controllers/movies/moviecollections');
+                    depends = 'controllers/movies/moviecollections';
                     break;
 
                 case 5:
-                    depends.push('controllers/movies/moviegenres');
+                    depends = 'controllers/movies/moviegenres';
                     break;
 
                 case 6:
-                    depends.push('scripts/searchtab');
+                    depends = 'scripts/searchtab';
```

Comment:
```suggestion
                    depends = 'scripts/searchtab';
                    break;
```

Adding a break to the last case is good practice, if someone ever adds another case and doesn't notice the last one is missing.

Reply:
Thanks